clinica/utils/filemanip.py

```diff
@@ -207,3 +207,40 @@ def extract_metadata_from_json(json_file, list_keys):
         file.close()
 
     return list_values
+
+
+def detect_cross_sectional_and_longitudinal_subjects(
+    all_subs: list, bids_dir: str
+) -> list:
+    """This function detects whether a subject as a longitudinal or cross sectionnal structure.
+    If it has a mixed structure, it will be considered cross sectionnal.
+
+    Parameters
+    ----------
+    all_subs: list
+        List containing all the names of the subjects.
+
+    bids_dir: str
+        Path to the bids directory.
```

Comment:
```suggestion
        Path to the bids directory.

```

Reply:
It's the same thing on my side. Do i need to change something ?

Reply:
I added an empty line to separate the sections.